contexts/v0.1.jsonld

```diff
@@ -4,11 +4,53 @@
     "id": "@id",
     "type": "@type",
     "lds": "https://w3c-ccg.github.io/lds-bbsbls2020/contexts/#",
-
-    "BbsBlsSignature2020": "lds:BbsBlsSignature2020",
+    "BbsBlsSignature2020": {
+      "@id": "https://w3c-ccg.github.io/lds-bbsbls2020/contexts/#BbsBlsSignature2020",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "sec": "https://w3id.org/security#",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "challenge": "sec:challenge",
+        "created": {
+          "@id": "http://purl.org/dc/terms/created",
+          "@type": "xsd:dateTime"
+        },
+        "domain": "sec:domain",
+        "signature": "sec:signature",
```

Comment:
If this signature type is using "proofValue" it doesn't need to also define `signature` right?

Reply:
yep I just removed it right as you commented.

Reply:
actually I removed `proofValue` because BbsBlsSignature2020 does use `signature`. BbsBlsSignatureProof2020 uses `proofValue`

Reply:
@dlongley currently we are using signature for `BbsBlsSignature2020` and proofValue for `BbsBlsSignatureProof2020` would you recommend consolidating to just using proofValue?

Reply:
I would recommend that both use `proofValue`, actually, `signature` should be deprecated -- it's from some older LD signature stuff, prior to work to unify signatures and other types of cryptographic proofs all under "Proofs". Therefore all new types should be using "proofValue", whether the proof type is a signature, a proof of existence, proof work, zkp, or whatever.

Reply:
This has been updated in the reference implementation now to use `proofValue` @haardikk21 can you update this context to have `proofValue` here?

Reply:
@tplooker addressed in https://github.com/w3c-ccg/ldp-bbs2020/pull/33/commits/0372c6d48ed483afc19d8a076d6c7d3c207e863c